### DISCUSSION TITLE
Switch between Assistants Mid Conversation

### DISCUSSION THREAD

**Discussion by airwindk** [danny-avila/LibreChat#4422] [Troubleshooting]
### What happened?

Switching between Azure Assistants mid conversation does not appear to reset the chat instance. If "Enable switching Endpoints mid-conversation" is turned off, I would expect to be re-directed to the landing page of the selected assistant which includes its description and conversation starters. However I see the conversation history associated with the assistant that was previously selected. 

### Steps to Reproduce

1. Turn off "Enable Switching Endpoints Mid-Conversation"
2. Create 2 dummy Azure Assistants
3. Ask Assistant 1 a question
4. Toggle to Assistant 2 (At this point I would expect to be redirected to Assistant 2's default page)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Endpoints would be OpenAI, Anthropic, Azure Assistants, etc.

What you're describing is expected behavior when switching assistants.

**Comment by airwindk** [comment]
Thank you for the quick reply and appreciate all your efforts with the project. Will add in a request for a new feature to enable switching mid conversation for assistants.
